include/crm/common/xml_compat.h

```diff
@@ -189,6 +189,9 @@ void copy_in_properties(xmlNode *target, const xmlNode *src);
 //! \deprecated Do not use
 void expand_plus_plus(xmlNode * target, const char *name, const char *value);
 
+//! \deprecated Do not use
```

Comment:
Recommend calling `crm_log_init()` or `crm_log_cli_init()` instead (those really serve as library-wide initializers). Note sbd calls `crm_xml_init()` but not `crm_xml_cleanup()`.